From 0e638e5aee9cf7cacfbe5bceedd16c0455423cae Mon Sep 17 00:00:00 2001
From: Sawan Roy <sawan.roy@trunexa.com>
Date: Mon, 11 Apr 2022 15:46:22 +0530
Subject: [PATCH] imx8mm: Allow parsing additional board specific

---
 init/ueventd.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/init/ueventd.cpp b/init/ueventd.cpp
index d2b503b..f4bcfad 100644
--- a/init/ueventd.cpp
+++ b/init/ueventd.cpp
@@ -287,9 +287,10 @@ int ueventd_main(int argc, char** argv) {
     // allow it to override everything.
     // TODO: cleanup platform ueventd.rc to remove vendor specific device node entries (b/34968103)
     auto hardware = android::base::GetProperty("ro.hardware", "");
+    auto board = android::base::GetProperty("ro.boot.board", "");
 
     auto ueventd_configuration = ParseConfig({"/system/etc/ueventd.rc", "/vendor/ueventd.rc",
-                                              "/odm/ueventd.rc", "/ueventd." + hardware + ".rc"});
+                                              "/odm/ueventd.rc", "/ueventd." + hardware + ".rc", "/vendor/ueventd."+ board + ".rc"});
 
     uevent_handlers.emplace_back(std::make_unique<DeviceHandler>(
             std::move(ueventd_configuration.dev_permissions),
-- 
2.7.4

